Manual/manJHUGenerator.tex

```diff
@@ -1614,16 +1632,21 @@ \subsubsection{"Higher order Graviton, spin-two", \texorpdfstring{$2^-_h$}{2-h}}
 \verb|a5=1,0 b8=1,0|
 \begin{thebibliography}{99}
 \bibitem{Gao:2010qx}
-Y.Y. Gao, A. V. Gritsan, Z.J. Guo, K. Melnikov, M. Schulze and N. V. Tran, ``Spin-Determination of Single-Produced Resonances at Hadron Colliders''. Phys. Rev. D \textbf{81}, 075022 (2010). arXiv:1001.3396 [hep-ph].
+Y.Y. Gao, A. V. Gritsan, Z.J. Guo, K. Melnikov, M. Schulze and N. V. Tran, ``Spin-Determination of Single-Produced Resonances at Hadron Colliders.'' Phys. Rev. D \textbf{81}, 075022 (2010). arXiv:1001.3396 [hep-ph].
```

Comment:
Just a minor comment: Citations should actually be in the form '".', not '."'. The previous version was correct. You can compare to how BibTeX handles the names in a few example papers.

Reply:
Seriously? This is the reason to hold this release?